input/blog/2017-09-13-cake-v0.22.0-released.md

```diff
@@ -0,0 +1,56 @@
+
+Updating Roslyn and unifying scripting engines has been a long-awaited improvement for many of you. This brings you C# 7 features (without need for `--experimental` switch), debugging with Mono (as we completely removed the old Mono scripting engine), and much, much more. It also allows us to ship Roslyn with Cake, so there's no need for Cake to actually download the Roslyn anymore. In order for us to update Roslyn, we also needed to change target framework to .NET 4.6.
+
+Tool and addin versioning when using preprocessor directives has been a issue which has caused many question. E.g. you may have noted that when changing version for a tool or addin, the old version was still used. Prior to this release, the only solution was to delete the tool/addin from the `tools` folder. This is now fixed as Cake supports side-by-side downloading of different versions of tools and addins. 
+
+Due to unpredictable results when downloading packages using `nuget.exe` tool/addin versioning was solved by adding a intermediate directory containing the version number. This intermediate directory can however cause `MAX_PATH` issues when downloading some specific packages. To address this issue, we implemented functionality in `Cake.NuGet` to allow Cake to download tools and addins without the need for `nuget.exe`. This functionality is currently optional through [configuration](../docs/fundamentals/configuration.md), and is the first step towards removing need for `nuget.exe` in Cake. Example usage to opt-in for the in-process NuGet client:
+```
+Cake.exe --NuGet_UseInProcessClient=true
+```
+The in-process NuGet client also gives the possibility to load assemblies from NuGet dependencies. This behaviour can be enabled either via [configuration](../docs/fundamentals/configuration.md) or directive. Example usage for loading dependencies:
+```
+Cake.exe --NuGet_UseInProcessClient=true --NuGet_LoadDependencies=true
+```
+or
+```
+#addin nuget:?package=Foo&LoadDependencies=true
+#addin nuget:?package=Foo.Bar&LoadDependencies=false
+```
+
+As 0.22.0 has several breaking changes, it will require addins to reference Cake.Core 0.22.0 or newer to be loaded, there's a "you're on your own anything can happen" setting to skip verification but we recommend to start targeting the newer version as soon as possible. Example usage to skip verification:
+```
+Cake.exe --Settings_SkipVerification=true
+```
+
+We have said it before, but here it comes again. Regardless of intentional or unintentional breaking changes, the possibility of new releases breaking something is why we always recommend you pin the version of Cake you're running, thus avoiding any unplanned broken builds. You can read more about pinning Cake version [here](https://cakebuild.net/docs/tutorials/pinning-cake-version).
```

Comment:
Maybe worth adding something like "We always try to avoid breaking changes, but we think these new features and improvements justify the friction, and we hope you will too!"

Just a thought...